weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1801,6 +1801,169 @@ def scorer_delete_v2(self, req: tsi.ScorerDeleteV2Req) -> tsi.ScorerDeleteV2Res:
         result = self.obj_delete(obj_delete_req)
         return tsi.ScorerDeleteV2Res(num_deleted=result.num_deleted)
 
+    def evaluation_create_v2(
+        self, req: tsi.EvaluationCreateV2Req
+    ) -> tsi.EvaluationCreateV2Res:
+        """Create an evaluation object.
+
+        Creates placeholder ops for evaluate, predict_and_score, and summarize methods.
+        """
+        # Create a safe ID for the evaluation
+        safe_name = object_creation_utils.make_safe_name(req.name)
+        evaluation_id = f"evaluation_{safe_name}"
+
+        # Create placeholder evaluate op
+        evaluate_op_req = tsi.OpCreateV2Req(
+            project_id=req.project_id,
+            name=f"{evaluation_id}_evaluate",
+            source_code=object_creation_utils.get_placeholder_evaluate_source(),
+        )
+        evaluate_op_res = self.op_create_v2(evaluate_op_req)
+        evaluate_ref = evaluate_op_res.digest
+
+        # Create placeholder predict_and_score op
+        predict_and_score_op_req = tsi.OpCreateV2Req(
+            project_id=req.project_id,
+            name=f"{evaluation_id}_predict_and_score",
+            source_code=object_creation_utils.get_placeholder_predict_and_score_source(),
+        )
+        predict_and_score_op_res = self.op_create_v2(predict_and_score_op_req)
+        predict_and_score_ref = predict_and_score_op_res.digest
+
+        # Create placeholder summarize op
+        summarize_op_req = tsi.OpCreateV2Req(
+            project_id=req.project_id,
+            name=f"{evaluation_id}_summarize",
+            source_code=object_creation_utils.get_placeholder_summarize_source(),
+        )
+        summarize_op_res = self.op_create_v2(summarize_op_req)
+        summarize_ref = summarize_op_res.digest
```

Comment:
All of these should be created ahead of time using the `ops` endpoint. Otherwise, when are they going to be set to their actual values?

Or maybe you are being proactive, since the eventual data model will no contain those op refs?

The problem is that this will yield some dummy ops in the UI, won't it?